### ISSUE TITLE
I launch the application and nothing happens

### ISSUE THREAD

**Issue by 1schlick** [archagon/sensible-side-buttons#62]

**Comment by adtyagrwl** [comment]
Yeah ive got the same issue. Not sure if its something to do with the M1 chip.

**Comment by zkendall** [comment]
I'm on 10.15.7 (no M1) and it doesn't work for me either. I think it used to.

**Comment by aideward** [comment]
same here I'm using m1

**Comment by aideward** [comment]
OH wait I jut found out why, looking at your system tray you have to enable it after you launched the app
now everything works fine
